script.js

```diff
@@ -0,0 +1,22 @@
+document.getElementById('check-btn').addEventListener('click', () => {
+  const userInput = document.getElementById('user-input').value;
+  const resultsDiv = document.getElementById('results-div');
+
+  if (userInput === '') {
+    resultsDiv.innerText('PleaSe provide a phone number!');
+    return;
+  }
+
+  const regex = /^(1\s?)?(\(\d{3}\)|\d{3})[-\s]?\d{3}[-\s]?\d{4}$/;
+
+  if (regex.test(userInput)) {
+    resultsDiv.innerText = `Valid US number: ${userInput}`;
+  } else {
+    resultsDiv.innerText = `Invalid US number: ${userInput}`;
+  }
+});
+
+document.getElementById('clear-btn').addEventListener('click', () => {
+  const resultsDiv = document.getElementById('results-div');
+  resultsDiv.innerText = '';
+});
```

Comment:
- [OPTIONAL] - Try to add your Event Listeners in while DOM content is loading, it helps to add all of your event listeners to be ready to the action right when your page loads and prevents any side effects below is an example to help you do that:

```javascript
document.addEventListener('DOMContentLoaded', () => {
  //In here add all of your addEventListener functions
  // don't forget to add the constants that you need in your event listeners here as the scope of this one is higher than other codes
  
  // Here add all of you code that belongs to event listeners
})
```